app/src/main/res/layout/topic_practice_subtopic.xml

```diff
@@ -32,12 +35,10 @@
       android:layout_height="wrap_content"
       android:button="@drawable/selector_checkbox"
       android:checked="@{isChecked}"
+      android:drawableStart="@android:color/transparent"
```

Comment:
Give min height as 48dp to pass accessibility test.

Reply:
But then the margin between the check boxes is getting increased. @veena14cs

Reply:
I am giving the checkbox height and width of 14dp. Will it be Okay by it?

Reply:
Don't give fixed width and height. Share screenshot after giving minHeight.

Reply:
![1](https://user-images.githubusercontent.com/36664384/76743778-282e4680-6799-11ea-9195-a5a250e70a6a.jpeg)
The margins are getting increased

Reply:
@veena14cs The screenshot when just minHeight of 48dp given and no margin
![1](https://user-images.githubusercontent.com/36664384/76744810-d8507f00-679a-11ea-8806-067a48094080.jpeg)